src/http.js

```diff
@@ -14,6 +14,17 @@ export const self = {
   mergeInQueryOrForm,
 };
 
+export class FetchResponseError extends Error {
```

Comment:
Please use this util (just for consistency) to create new Error Type: https://github.com/swagger-api/swagger-js/blob/3b5cc18fd7c738f3954609b0820459ef57e15c11/src/specmap/lib/create-error.js#L5

Here is an example how new Error types are created using this util: https://github.com/swagger-api/swagger-js/blob/57a2436a352997e9426ee795f2a21d2075ae251d/src/execute/index.js#L17-L16

The util handles `Error.captureStackTrace` static method of V8. More about it here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#custom_error_types